doc/chromebooks/installation/luks-installation.md

```diff
@@ -101,19 +92,20 @@ _Note. some newer chromebooks can boot 64mb partitions but for now let's stick t
 
 - you can edit partitions graphically using gparted(```sudo apt install gparted```)
 ![gparted](./assets/luks/gparted.png)
-- or terminal using fdisk
+- or in terminal using fdisk
 ```
 fdisk /dev/${disk}
 ```
-- or more graphically cfdisk (```sudo apt install util-linux```)
-```
+![fdisk](./assets/luks/fdisk.png)
+- or more graphically but still in terminal with cfdisk (```sudo apt install util-linux```)
 cfdisk /dev/${disk}
-```
 ![cfdisk](./assets/luks/cfdisk.png)
 
 _Note. you don't need to format them or select specyfic type, just make them correct size we will format them later_
 
-5. make filesystems
+_Warning. this images are for software look showcase and to make it easier for you to chose the most comfortable one for you <3, partition your device the way described in the guide, not the way presented on the images_
+
+1. make filesystems
 
 format boot partition
 ```
```

Comment:
two lines lower there is a typo lable - label ... i'm adding the note here as the below lines are not changed, so i cannot add the note there directly easily ... i aslo spotted that typo a bit further down once more

Reply:
https://github.com/hexdump0815/imagebuilder/pull/247/commits/5a844d546cfa9755c96cfc63d438b1b05d82c483